packages/components/src/button/stories/index.tsx

```diff
@@ -0,0 +1,104 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+import type { ReactNode } from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	formatBold,
+	formatItalic,
+	link,
+	more,
+	wordpress,
+} from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import './style.css';
+import Button from '..';
+
+const meta: ComponentMeta< typeof Button > = {
+	title: 'Components/Button',
+	component: Button,
+	argTypes: {
```

Comment:
One last thing for the `argTypes`, let's make this small override so the `href` prop doesn't look like a required prop in the Storybook props table:

```suggestion
	argTypes: {
		// Overrides a limitation of the docgen interpreting our TS types for this as required.
		href: { type: { name: 'string', required: false } },
```

Reply:
Thanks, committed in https://github.com/WordPress/gutenberg/pull/46997/commits/07b70022b95ef031a4e3194f49fda1ba64ef4266